csharp/src/Ice/IObject.cs

```diff
@@ -6,30 +6,28 @@
 using System;
 using System.ComponentModel;
 using System.Diagnostics;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Ice
 {
-    /// <summary>
-    /// Interface for incoming requests.
-    /// </summary>
-    public interface IRequest
-    {
-        /// <summary>
-        /// Returns the {@link Current} object for this the request.
-        /// </summary>
-        /// <returns>The Current object for this request.</returns>
-        Current? GetCurrent();
-    }
-
     ///<summary>The base interface for all servants.</summary>
     public interface IObject
     {
-        public Task<OutputStream>? Dispatch(Incoming inS, Current current)
+        /// <summary>Dispatches a request on this servant.</summary>
+        /// <param name="istr">The <see cref="InputStream"/> that holds the request frame. It is positionned at
+        /// the start of the request's payload, just after starting to read the encapsulation.</param>
+        /// <param name="current">The current parameter holds decoded header data and other information about the
+        /// current request.</param>
+        /// <returns>A value task that provides the response frame for the request. Null means the request is a oneway
```

Comment:
return comment is outdated, still refer to Null.

Reply:
`positionned` should be `positioned`